packages/core/describers/transactions/lib/tokenTxDescribers.js

```diff
@@ -81,15 +81,15 @@ export const FUNCTION_APPROVE = new TransactionDescriber(
 
 export const FUNCTION_REQUIRE_TIME = new TransactionDescriber(
   findFunctionABI(AssetDonatorABI, 'sendTime'),
-  ({ tx, block }, context, { abi }) => {
+  ({ tx, block }) => {
     const symbol = TIME
     const transferAmount = new Amount(1000000000, symbol)
-    const path = `tx.${abi.contractName}.sendTime`
+    const path = `tx.${ERC20DAODefaultABI.contractName}.sendTime`
```

Comment:
Comment for the line 86:
please create new constant for the  1000000000 and append comment like "10 TIME, value hardcoded somewhere in Solidity"